src/lucky_record/charms/int32_extensions.cr

```diff
@@ -18,7 +18,7 @@ struct Int32
     end
 
     def self.to_db(value : Int32)
-      value.to_s
+      value
```

Comment:
This change broke a lot of tests that expected Int32 to be a string.

Reply:
You’re probably right but I think it’s be better to handle that separately. Could you open an issue about removing the to_s?